--- /home/pi/ntpviz	2023-07-20 18:49:34.119503877 -0500
+++ ntpviz	2023-07-20 22:12:40.000000000 -0500
@@ -1531,7 +1531,7 @@
     if 's' == args.width:
         # fit in 1024x768 browser
         # in 2016 this is 22% of all browsers
-        args.img_size = '1000,720'
+        args.img_size = '800,420'
     elif 'l' == args.width:
         # fit in 1920x1080 browser
         args.img_size = '1850,1000'
@@ -1732,51 +1732,21 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<link rel="shortcut icon" href="favicon.ico">
+<link rel="shortcut icon" href="/favicon.ico">
 <meta charset="UTF-8">
 <meta http-equiv="refresh" content="1800">
 <meta name="expires" content="0">
 <title>%(title)s</title>
-<style>
-dt {
-    font-weight: bold;
-    margin-left: 20px;
-}
-dd {
-    margin-top: 4px;
-    margin-bottom: 10px;
-}
-table {
-    text-align: right;
-    width: 1300px;
-    border-collapse: collapse;
-}
-thead {
-    font-weight: bold;
-}
-tbody tr {
-    vertical-align: top;
-}
-tbody tr:nth-child(6n+4),
-tbody tr:nth-child(6n+5),
-tbody tr:nth-child(6n+6) {
-    background-color: rgba(0,255,0,0.2);
-}
-.section {
-    color: #000000;
-    text-decoration: none;
-}
-.section .site-title:visited {
-    color: #000000;
-}
-</style>
+<link rel="stylesheet" type="text/css" href="/graphs.css" />
 </head>
 <body>
-<div style="width:910px">
+<div id='graphid'>
+<div id='headerimage'>
 <a href='https://www.ntpsec.org/'>
-<img src="ntpsec-logo.png" alt="NTPsec" style="float:left;margin:20px 70px;">
+<img src="/ntpsec-logo.png" alt="NTPsec">
 </a>
-<div>
+</div>
+<div class='headerblock'>
 <h1 style="margin-bottom:10px;">%(title)s</h1>
 <b>Report generated:</b> %(report_time)s <br>
 ''' % locals()
@@ -1806,7 +1776,7 @@
 <span style="color:red;font-weight:bold;">Warning: plots clipped</span><br>
 """
 
-    index_header += '</div>\n<div style="clear:both;"></div>'
+    index_header += '</div>\n'
 
     index_trailer = '''\
 <h2>Glossary:</h2>
@@ -1902,7 +1872,7 @@
 <a href="https://docs.ntpsec.org/latest/ntpviz.html">
 ntpviz</a>, part of the <a href="https://www.ntpsec.org/">NTPsec project</a>
 </div>
-<div style="float:left;margin-left:350px;">
+<div style="float:right;">
     <a href="https://validator.w3.org/nu/">
     <img src="https://www.w3.org/html/logo/downloads/HTML5_Logo_32.png"
         alt="html 5">
@@ -1930,7 +1900,7 @@
         try:
             header_file = open(header, 'r', encoding='utf-8')
             header_txt = header_file.read()
-            index_buffer += '<br>\n' + header_txt + '\n'
+            index_buffer += '\n' + header_txt + '\n'
         except IOError:
             pass
 
